File: models/task.go
package models

type Task struct {
	TaskId          int        `json:"taskId"`
	ProjectId       int        `json:"projectId"`
	Name            string     `json:"name"`
	Creator         int        `json:"creator"`
	CreatorName     string     `json:"creatorName"`
	CreatorIconPath string     `json:"creatorIconPath"`
	Assign          int        `json:"assign"`
	AssignName      string     `json:"assignName"`
	AssignIconPath  string     `json:"assignIconPath"`
	Status          TaskStatus `json:"status"`
	Deadline        string     `json:"deadline"`
	LimitDate       int        `json:"limitDate"`
	DeadlineMD      string     `json:"deadlineMD"`
	Description     string     `json:"description"`
	CreateDate      int64      `json:"createDate"`
	Adopted         int64      `json:"updateDate"`
	IsDoing         bool       `json:"is_doing"`
}

File: www/src/assets/scripts/model/api/task/Task.ts
export default class Task {
    public taskId: number
    public projectId: number
    public name: string
    public creator: number
    public creatorName: string
    public creatorIconPath: string
    public assign: number
    public assignName: string
    public assignIconPath: string
    public status: number
    public deadline: string
    public limitDate: number
    public deadlineMD: string
    public description: string
    public createDate: string
    public updateDate: string
    public isDoing: boolean

    public fromAny(data: any) {
        this.taskId = data["taskId"]
        this.projectId = data["projectId"]
        this.name = data["name"]
        this.creator = data["creator"]
        this.creatorName = data["creatorName"]
        this.creatorIconPath = data["creatorIconPath"]
        this.assign = data["assign"]
        this.assignName = data["assignName"]
        this.assignIconPath = data["assignIconPath"]
        this.status = data["status"]
        this.deadline = data["deadline"]
        this.limitDate = data["limitDate"]
        this.deadlineMD = data["deadlineMD"]
        this.description = data["description"]
        this.createDate = data["createDate"]
        this.updateDate = data["updateDate"]
        this.isDoing = data["is_doing"]
    }

    public toJson(): string {
        return JSON.stringify(this)
    }
}